src/pipelines/hospitalizations/config.yaml

```diff
@@ -133,6 +132,16 @@ sources:
       skip: True
       metadata_query: key.str.match("GB_SCT.*")
 
+  # Data sources for GB level 1
+  - name: pipelines.hospitalizations.gb_authority.UKL1DataSource
+    fetch:
+      - url: "https://api.coronavirus.data.gov.uk/v1/data?filters=areaType=overview&structure=%7B%22areaType%22:%22areaType%22,%22areaName%22:%22areaName%22,%22areaCode%22:%22areaCode%22,%22date%22:%22date%22,%22newAdmissions%22:%22newAdmissions%22,%22cumAdmissions%22:%22cumAdmissions%22%7D&format=csv"
```

Comment:
Would be better to use the API directly instead of encoding the params in the URL string like here. See https://github.com/GoogleCloudPlatform/covid-19-open-data/blob/main/src/pipelines/epidemiology/gb_authority.py for an example of how to do this.